Project 5 For John Beverly

```diff
@@ -59,345 +59,388 @@ Group 2 contains Emanuel, Katie, Josh V (commonly know as V by group member), Ca
+
+sh:datatype xsd:string ; 
 
 ] .
 
+2.	Constraint for Symmetry 
+@prefix sh: <http://www.w3.org/ns/shacl#> .
 
-ex:lineRegistrationOrder
+@prefix ex: <http://example.com/> .
 
-a sh:Order ;
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
-sh:datatype xsd:integer ;
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-sh:path ex:addedAt ;
 
-sh:ascending true ;
+ex:SymmetryShape
 
-sh:uniqueMembers true .
+a sh:NodeShape ;
 
+sh:targetClass ex:relation ;
 
-ex:RegistrationDateShape
-  
-  a sh:NodeShape ;
-  
-  sh:targetClass ex:RegistrationDate ;
-  
-  sh:property [    
-  
-  sh:path ex:RegistrationDate ;    
-  
-  sh:datatype xsd:string ;   
- 
- Sh:description
- 
- "US Date - format - MM/DD/YYYY" ;     
+sh:property [    
 
-sh:minCount 1 ;   
+sh:path rdf:value ;    
 
-sh:maxCount 1 ;   
+sh:minCount 1 ;    
 
-sh:name "RegistrationDate" ;    
+sh:maxCount 1 ;    
 
-sh:pattern "[1-12]/[1-31]/[1990-2020]”" ;  
+sh:datatype xsd:string 
 
 ] .
 
+3.	Constraint Symmetry with a check things not being symmetric 
+@prefix sh: <http://www.w3.org/ns/shacl#> .
 
+@prefix ex: <http://example.com/> .
 
-ex:lineRegistrationDateOrder
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
-a sh:Order ;
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-sh:datatype xsd:integer ;
+ex:SymmetryShape
 
-sh:path ex:RegistrationAt ;
+a sh:NodeShape ;
 
-sh:ascending true ;
+sh:targetclass ex:relation ;
 
-sh:uniqueMembers true .
+sh:property [    
 
+sh:path rdf:value ;    
 
-ex:RegistrationDateANDlineRegistrationOrder
+sh:minCount 2 ;    
 
-a sh:NodeShape ;
+sh:datatype xsd:string ;    
 
-sh:and (ex:RegistrationDateShape ex:lineRegistrationDateOrder) ;
+sh:nodeKind sh:Literal ;    
 
-sh:property [    
+sh:message "The relation does not have symmetric values"  
 
-sh:path rdf:type ;    
+] ;
 
-sh:datatype xsd:integer ;   
+sh:property [    
 
-sh:node ex:ValidationResult ;   
+sh:path [ 
 
-sh:resultSeverity sh:Violation ;   
+sh:inversePath rdf:value 
 
-sh:sourceConstraintComponent ex:RegistrationDate ;    
+] ;
 
-sh:name "RegistrationDateANDlineRegistrationOrder" ; 
+sh:minCount 2 ;
 
-sh:focusNode :lineRegistrationDateOrder ;   
+sh:datatype xsd:string ;
 
-sh:value :integer ;   
+sh:nodeKind sh:Literal ;
 
-sh:resultPath schema:RegistrationDate ; 
+sh:message "The relation does not have symmetric values"
 
 ] .
 
-
-
-2nd Constraints begins below
-
+4.	Constraint for Reflexivity 
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 
-@prefix ex: <http://example.org/> .
+@prefix ex: <http://example.com/> .
+
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-ex:SpatialLocationShape
+
+ex:ReflexiveShape
 
 a sh:NodeShape ;
 
-sh:targetClass ex:Location ;
+sh:targetClass ex:relation ;
+
+sh:property [    
+
+sh:path rdf:value ;    
 
-sh:property [                  
+sh:minCount 1 ;    
 
-sh:path ex:spatialcontainment ;              
+sh:maxCount 1 ;    
```

Comment:
So we are saying that this class can have at most one rdf:value, right? How is this entailing reflexivity?